extensionsGallery-insider.json

```diff
@@ -5336,6 +5336,67 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "105",
+					"extensionName": "catppuccin-vsc",
+					"displayName": "Catppuccin for Azure Data Studio",
+					"shortDescription": "🦌 Soothing pastel theme for Azure Data Studio",
+					"publisher": {
+						"displayName": "Catppuccin",
+						"publisherId": "Catppuccin",
+						"publisherName": "Catppuccin"
+					},
+					"versions": [
+						{
+							"version": "3.8.0",
+							"lastUpdated": "11/30/2023",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://github.com/catppuccin/vscode/releases/tag/v3.8.0"
```

Comment:
If you wanted to you could still leave this as a direct link to the VSIX though - it'll just download it to the users machine where they'll have to install it manually. But that'll save a few clicks. Let me know if you'd prefer to stay with this though.

Reply:
Yup!